package.json

```diff
@@ -34,7 +38,7 @@
     "@babel/eslint-parser": "^7.22.15",
     "@modyfi/vite-plugin-yaml": "^1.0.4",
     "@trivago/prettier-plugin-sort-imports": "^1.4.4",
-    "@tsconfig/node20": "^20.1.2",
+    "@tsconfig/node18": "^18.2.2",
```

Comment:
Il y a beaucoup d'autres choses dans ce commit en plus de ce changement de version minimale !

Le reste fait-il partie de cette PR ? Et si oui 1) en quoi est-ce lié ? 2) faut-il modifier la description de la PR ?

Reply:
Everything is linked to type-checks, these are just stubs for linters to check them properly. including this line (excepting the "engines" a bit above).

Reply:
Je parlais des autres fichiers dans le [commit](https://github.com/ecolabdata/ecospheres-front/pull/170/commits/ead82f7008649ed1b9eb23119d86f7c2ff087c58), pas des lignes dans le fichier. 
En particulier : comment l'ajout de `vite.ts.config` est-il lié au changement de version minimale ?

Reply:
Le fichier existait déjà, il a juste changé de nom (de .js à .ts).

Reply:
ok, c'est plus clair maintenant :)